### ISSUE TITLE
[BUG] Taxes and charges  not populating 

### ISSUE THREAD

**Issue by govindsmenokee** [frappe/erpnext#18446]
Taxes and charges child table does not populate based on tax rule defined. Issue is there for quotation, sales invoice 
Version 11 and 12 are effected

**Comment by rohitwaghchaure** [comment]
@govindsmenokee Have you enabled "Automatically Add Taxes and Charges from Item Tax Template" in the accounts settings? If not please enable and check

**Comment by govindsmenokee** [comment]
@rohitwaghchaure Yes I have. Also the template field gets populated but the actual table does not

**Comment by rohitwaghchaure** [comment]
@govindsmenokee Please share the GIF

**Comment by nasirrr** [comment]
Same issue is occuring in V13.16.0 as well.
